pyipmi/interfaces/rmcp.py

```diff
@@ -350,7 +360,7 @@ class Rmcp(object):
     _session = None
 
     def __init__(self, slave_address=0x81, host_target_address=0x20,
-                 keep_alive_interval=1):
+                 keep_alive_interval=1, check_sdu_length=True):
```

Comment:
I think a better solution would be a configuration dict for quirks like this. With that we can avoid having more and more arguments for the interfaces and it is more generic.

e.g. `quirks_cfg={'rmcp_ignore_sdu_length': True}`

Then we can check for that with `if 'rmcp_ignore_sdu_length' in quirks_cfg:`

What do you think?

I agreed please add a doc string for the Rmcp object and describe the usage of that quirk.

Reply:
Yeah you're right, it is more "future proof" this way :+1: 
I will add doc string and make the corresponding modifications